runtime/src/extensions.rs

```diff
@@ -306,67 +306,66 @@ mod tests {
 	#[ignore]
 	fn dispatch_balance_transfer_from_contract_works() {
 		new_test_ext().execute_with(|| {
-            let _ = env_logger::try_init();
-
-            let (wasm_binary, _) = load_wasm_module::<Runtime>("../contracts/pop-api-examples/balance-transfer/target/ink/pop_api_extension_demo.wasm").unwrap();
-
-            let init_value = 100 * UNIT;
-
-            let result = Contracts::bare_instantiate(
-                ALICE,
-                init_value,
-                GAS_LIMIT,
-                None,
-                Code::Upload(wasm_binary),
-                function_selector("new"),
-                vec![],
-                DEBUG_OUTPUT,
-                pallet_contracts::CollectEvents::Skip,
-            )
-                .result
-                .unwrap();
-
-            assert!(
-                !result.result.did_revert(),
-                "deploying contract reverted {:?}",
-                result
-            );
-
-            let addr = result.account_id;
-
-            let function = function_selector("transfer_through_runtime");
-            let value_to_send: u128 = 10 * UNIT;
-            let params = [function, BOB.encode(), value_to_send.encode()].concat();
-
-            let bob_balance_before = Balances::free_balance(&BOB);
-            assert_eq!(bob_balance_before, INITIAL_AMOUNT);
-
-            let result = Contracts::bare_call(
-                ALICE,
-                addr.clone(),
-                0,
-                Weight::from_parts(100_000_000_000, 3 * 1024 * 1024),
-                None,
-                params,
-                DEBUG_OUTPUT,
-                pallet_contracts::CollectEvents::Skip,
-                pallet_contracts::Determinism::Enforced,
-            );
-
-            if DEBUG_OUTPUT == pallet_contracts::DebugInfo::UnsafeDebug {
-                log::debug!(
-                    "Contract debug buffer - {:?}",
-                    String::from_utf8(result.debug_message.clone())
-                );
-                log::debug!("result: {:?}", result);
-            }
-
-            // check for revert
-            assert!(!result.result.unwrap().did_revert(), "Contract reverted!");
-
-            let bob_balance_after = Balances::free_balance(&BOB);
-            assert_eq!(bob_balance_before + value_to_send, bob_balance_after);
-        });
+			let _ = env_logger::try_init();
+
+			let (wasm_binary, _) = load_wasm_module::<Runtime>(
+				"../pop-api/examples/balance-transfer/target/ink/pop_api_extension_demo.wasm",
```

Comment:
I think we should just include the resulting wasm artifact for now, so that the ignore flags can be removed from the extension tests and they therefore run during CI checks.

We can look to automate this with a build.rs build script.